.travis.yml

```diff
@@ -3,6 +3,9 @@ language: node_js
 node_js:
   - '6'
 
+before_install:
+  - set -e
```

Comment:
wouldn't it be cool to have the `-x` enabled if the env var `DEBUG` is equal to one? Or aren't we expecting much bash logic in the the CI?

Reply:
@jackTheRipper nope, we're not. This is purely execution of commands without logic.

But can you explain a little bit more in detail what `-x` would allow us to do?

Reply:
Oh well, Google is our friend, here's some more info: https://stackoverflow.com/questions/36273665/what-does-set-x-do

Reply:
@jackTheRipper however, the commands executed at CI level are already printed on Travis CI.

Nevertheless, I'm all open for adding it at a later stage if you see a benefit on it!

Reply:
@mAiNiNfEcTiOn sorry for the late reply. I offered to add the `-x` flag in debug mode for having the details of the bash commands. But such as what I said in my original comment it makes sense only if we do have some logic in bash (which seems to be not the case for this project currently)